conf/messages

```diff
@@ -1,5 +1,5 @@
-error.postcode.invalid = Please enter a valid postcode
-error.postcode.blacklisted = This postcode is blocked and cannot be used
+error.postcode.invalid = You have entered an invalid postcode
+error.postcode.blacklisted = You can't use the postcode you've entered
```

Comment:
Did this content change come from Charlie? (the old message seems clearer to me but if the change came from Charlie then it's fine)